File: class13/cpp/AP1301-Calculadora.cpp
#include <iostream>
int main() {
    int op; double a, b;
    do {
        std::cout << "1-Somar 2-Sub 3-Mult 4-Div 5-Sair\nOpcao: ";
        std::cin >> op;
        if (op >= 1 && op <= 4) {
            std::cout << "N1: "; std::cin >> a;
            std::cout << "N2: "; std::cin >> b;
        }
        switch (op) {
            case 1: std::cout << a+b << std::endl; break;
            case 2: std::cout << a-b << std::endl; break;
            case 3: std::cout << a*b << std::endl; break;
            case 4: if(b!=0) std::cout << a/b << std::endl; break;
        }
    } while (op != 5);
    return 0;
}

File: class13/java/AP1301-Calculadora.java
import java.util.Scanner;
public class Calc {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        while (true) {
            System.out.println("1-Somar 2-Sub 3-Mult 4-Div 5-Sair");
            int op = sc.nextInt();
            if (op == 5) break;
            System.out.print("N1: "); double a = sc.nextDouble();
            System.out.print("N2: "); double b = sc.nextDouble();
            double r = 0;
            switch (op) {
                case 1: r = a+b; break; case 2: r = a-b; break;
                case 3: r = a*b; break; case 4: if(b!=0) r=a/b; else System.out.println("Erro"); break;
            }
            System.out.println("Resultado: " + r);
        }
    }
}
